pkgs/top-level/all-packages.nix

```diff
@@ -6676,6 +6676,7 @@ with pkgs;
 
   meilisearch = callPackage ../servers/search/meilisearch {
     inherit (darwin.apple_sdk.frameworks) Security;
+    inherit pkgs;
```

Comment:
pkgs shouldn't really be inherited ever. Can we just inherit what is needed instead or use callPackage?

Reply:
I did this because crate2nix uses pkgs as an argument.
Do you mean to say that in this case, the solution should be
```
pkgs = { inherit buildRustPackage ... ; };
```
having a look at everything that crate2nix needs and put them there.
(I don't know how long that list is yet, I don't mind having a look)